frontend/lib/src/components/elements/Favicon/Favicon.test.tsx

```diff
@@ -42,69 +42,42 @@ describe("Favicon element", () => {
   const endpoints = mockEndpoints({ buildMediaURL: buildMediaURL })
 
   it("sets the favicon in the DOM", () => {
-    handleFavicon(
-      "https://some/random/favicon.png",
-      jest.fn(),
-      endpoints,
-      true
-    )
+    getFaviconUrl("https://some/random/favicon.png", endpoints)
     expect(buildMediaURL).toHaveBeenCalledWith(
       "https://some/random/favicon.png"
     )
+    overwriteFavicon("https://mock.media.url/")
     expect(getFaviconHref()).toBe("https://mock.media.url/")
   })
 
   it("accepts emojis directly", () => {
-    handleFavicon("🍕", jest.fn(), endpoints, true)
+    getFaviconUrl("🍕", endpoints)
```

Comment:
I think these tests (also the other tests below) need a bit of refactoring since calling `getFaviconUrl ` here doesn't really add anything to the test anymore. So, it probably needs some independent tests for `getFaviconUrl` and some for `overwriteFavicon`.